### ISSUE TITLE
v2: [BUG] the { hamburger > location wizards } page does not scroll

### ISSUE THREAD

**Issue by gnomicosuw** [aquariumbio/aquarium#506]
repro:

1. log into aquarium as user with with admin privileges
2. navigate to { hamburger > location wizards } page
result: the page does not scroll (makes it kind of challenging to edit a location wizard that is below the fold)

(see screenshot - notice there is no scrollbar)

<img width="1328" alt="Screen Shot 2020-12-21 at 2 17 57 PM" src="https://user-images.githubusercontent.com/69167459/102827600-b45c4280-4397-11eb-805a-e71c19c8da8d.png">

note: you can add overflow-y:scroll to the < div layout="row" class="layout-row" style="overflow-y:scroll" > div in the view and get the content to scroll.  It might be nicer to scroll the table contents and keep the title, but that also affects the vertical space.  And since this is all probably going to change later I would do whatever is easiest. (It has been this way for a long time and as far as I know no one has complained).

Here's another screenshot with the overflow setting (notice the scrollbar)

<img width="1328" alt="Screen Shot 2020-12-21 at 2 32 59 PM" src="https://user-images.githubusercontent.com/69167459/102828416-8546d080-4399-11eb-8cb1-8f9388cca5e4.png">

**Comment by bjkeller** [comment]
Is this problem present in either the master or staging branch?

**Comment by gnomicosuw** [comment]
It is in both.

**Comment by bjkeller** [comment]
OK, let's update this in tech-handoff

**Comment by gnomicosuw** [comment]
There is a whole bunch of Angular and/or React stuff on this page which I did not want to try to decipher, so I just hacked in a solution.
- added a variable { @scroll = true }
- added inline styling { style="overflow-y:scroll" } if @scroll is true

Note that the "Budgets" page does scroll, so there is probably some setting buried somewhere.  But the hack works.

I checked it in under the branch "scroll" off of "master". I will submit a PR.
